### PR TITLE
docs(beginner-guide): Final Approach and ILS Llanding

### PR DESCRIPTION
**Pull request by frankkopp**
- [x] Basic structure
- [x] Intercepting the ILS Localizer
- [x] Intercepting the ILS Glidescope
- [x] Preparation and Checklist for Landing
- [x] Landing
- [x] Vacate Runway

Needs pilot review for accuracy.
- [x] Pilot Review (2 done)

### CODE REVIEW

**Comment by tareksabet** [comment]
LGTM

File: docs/beginner-guide/landing.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
+# Final Appraoch and ILS Landing
```
Comment (Valastiri) [inline docs/beginner-guide/landing.md]:
```suggestion
# Final Approach and ILS Landing
```

File: docs/beginner-guide/landing.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+We hold the attitude of the aircraft until it settles on the ground. **Do not push the sidestick forward (nose down) once flared.** We let the aircraft settle to the runway while holding the pitch.
+
+**Practice this!!**
+
+On touchdown of the main gear the `Speed Brakes` will deploy automatically (we have armed them earlier) and we pull the thrust levers into the reverse position.
+
+![Reverse Thrust](../assets/beginner-guide/landing/Thrustlevers_Revers.png "Reverse Thrust")
+
+We let the front gear settle gently on the runway (don't slam it down) and hold the center line of the runway while we are reducing speed. The speed reduction should be monitored on the `PFD` speedband and the speed trend arrow.
```
Comment (Valastiri) [inline docs/beginner-guide/landing.md]: is speedband one word? or two words? "speed bank". If Airbus moniker leave as is

**Review by Valastiri — CHANGES_REQUESTED** [review]
Single typo + one suggestion.

**Review by Valastiri — APPROVED** [review]

**Pull request merged**